src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,24 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
```

Comment:
```suggestion
export const PostInfo = ({ post: { title } }) => (
```
You can do the next level of destructuring assignment
Or just do it below

Same in other components